spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb

```diff
@@ -428,6 +428,7 @@ def parse_entities(namespace_pod, namespace_config)
                                    :resource_version              => '172339',
                                    :labels                        => [],
                                    :objects                       => [],
+                                   :type                          => "ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate",
```

Comment:
@lfu @cben have you any idea what this value was before? `nil`? or as what class were these loaded?